lib/vicadmin/validate.go

```diff
@@ -65,6 +67,8 @@ const (
 	GoodStatus     = template.HTML(`<i class="icon-ok"></i>`)
 	BadStatus      = template.HTML(`<i class="icon-attention"></i>`)
 	DefaultVCHName = ` `
+	HTTPSProxy     = "--https-proxy"
```

Comment:
These should be unexported, i.e. `httpsProxy` and `httpProxy` unless they're being used outside this package. 

Actually, it would be nice to leave these exported and use them in https://github.com/vmware/vic/blob/master/cmd/vic-machine/create/create.go (lines 453 and 461) so we can consolidate these variables. Check with @emlin on this approach.